bee-ledger/src/workers/pruning/batch.rs

```diff
@@ -42,8 +41,8 @@ pub struct Edge {
     pub to_child: MessageId,
 }
 
-pub fn prune_confirmed_data<S: StorageBackend>(
-    tangle: &Tangle<S>,
+pub fn batch_prunable_confirmed_data<S: StorageBackend>(
+    _tangle: &Tangle<S>,
```

Comment:
Do you plan on using this again in the future? Otherwise we could leave it out.

Reply:
I removed it :smile: